### PR TITLE
fix: [CVS-8848]: added endpoint /change-event/monitored-service-timeline

### PR DESCRIPTION
**Pull request by deepesh-ui**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by deepesh-ui** [comment]
trigger cypress

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.tsx [hunk 1]
```diff
@@ -57,8 +75,8 @@ export default function ChangeTimeline(props: ChangeTimelineProps): JSX.Element
   }, [endTime, selectedTimePeriod?.value, hideTimeline, startTime])
 
   useEffect(() => {
-    cancel()
-    refetch({
+    changeEventTimelineCancel()
+    changeEventTimelineRefetch({
```
Comment (knagurski) [inline src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.tsx:79]: Just wondering... but doesn't a `refetch` cancel by default?

File: src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.tsx [hunk 2]
```diff
@@ -81,6 +99,35 @@ export default function ChangeTimeline(props: ChangeTimelineProps): JSX.Element
...
+        projectIdentifier,
+        environmentIdentifier: Array.isArray(environmentIdentifier) ? environmentIdentifier[0] : environmentIdentifier,
+        serviceIdentifier: Array.isArray(serviceIdentifier) ? serviceIdentifier[0] : serviceIdentifier,
+        changeSourceTypes: changeSourceTypes || [],
+        duration: duration?.value as TimePeriodEnum,
+        endTime: Date.now()
+      }
+    })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [duration])
```
Comment (knagurski) [inline src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.tsx:117]: Worth adding the deps here
```suggestion
  }, [accountId, orgIdentifier, projectIdentifier, environmentIdentifier, serviceIdentifier, changeSourceTypes, duration])
```

File: src/modules/85-cv/components/ChangeTimeline/__tests__/ChangeTimeline.test.tsx [hunk 3]
```diff
@@ -30,6 +30,23 @@ jest.mock('services/cv', () => ({
...
+            Alert: []
+          }
+        }
+      },
+      refetch: jest.fn(),
+      error: null,
+      loading: false,
+      cancel: jest.fn()
+    }
   })
```
Comment (knagurski) [inline src/modules/85-cv/components/ChangeTimeline/__tests__/ChangeTimeline.test.tsx:50]: You can simplify this a wee bit
```suggestion
  useGetMonitoredServiceChangeTimeline: jest.fn().mockReturnValue({
    data: {
      resource: {
        categoryTimeline: {
          Deployment: [],
          Infrastructure: [],
          Alert: []
        }
      }
    },
    refetch: jest.fn(),
    error: null,
    loading: false,
    cancel: jest.fn()
  })
```

**Comment by deepesh-ui** [comment]
retrigger eslint

File: src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.tsx (hunk 1 above)
Comment (deepesh-ui) [reply src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.tsx:79]: Agree , i am also under the impression that it does cancel by default . but for the scope of this PR not making change to any existing logic.

**Comment by deepesh-ui** [comment]
retrigger eslint

**Comment by deepesh-ui** [comment]
retrigger jest

**Review by vasuki-kalluri — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**